README.md

```diff
@@ -10,6 +10,12 @@ but also make certain assumptions that you may not want to impose on your module
 If you are making changes to the plugins, please see the [internal docs](https://internal.labkey.com/Handbook/Dev/wiki-page.view?name=gradlePlugins) for more information
 on how to do that, including how to develop and test locally and the versioning information.
 
+## Release Notes
+### TBD
+*Released*: TBD
+(Earliest compatible LabKey version: 21.3)
+* Enable build, deploy, and distributions without `server` repository
```

Comment:
Please add more details here about the changes for taglibs and tomcat jars and the addition of the Standalone plugin.